### PR TITLE
refactor: A380 batt display change

### PR DESCRIPTION
**Pull request by Steveveepee**
Fixes https://github.com/flybywiresim/aircraft/issues/9002

## Summary of Changes
Changed behaviour of the Batt Voltage Display when OFF Selected to be blank rather than 00.0V

## Additional context
I'm sure its blanked in OFF. 

Discord username: Shrike

## Testing instructions

1. make sure Display is blank when turned to OFF position
2. it should show values when switching to any non OFF position and they should update one digit at a time

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Comment by Steveveepee** [comment]
Switched to pull on Master

File: fbw-a380x/src/systems/instruments/src/BAT/index.tsx [hunk 1]
```diff
@@ -17,11 +18,24 @@ const BatRoot = () => {
 
     // mapping of knob (lvar) values to battery numbers to allow easy lvar and model values
     const batteryMap = [4, 3, 0, 1, 2]; // ESS, APU, OFF, BAT1, BAT2
```
[the remark this answers is not in the document]
Comment (frankkopp) [reply fbw-a380x/src/systems/instruments/src/BAT/index.tsx:20]: The enum is indeed nice but your comment about this comment is wrong ;) - this is the mapping of the knob position (order of values) to the bat number (values) and therefore the comment is correct. And even if a comment would be wrong once in while it is still no argument for not commenting code. It is simple good practice

**Comment by Steveveepee** [comment]
Can this be merged before release?

**Review by frankkopp — APPROVED** [review]

**Comment by frankkopp** [comment]
> Can this be merged before release?

I approved it now - but needs at least one QA test to be considered for merging. 

Btw. do you have a reference (maybe foto) that the display is off or shows "---" when in off position? 
(seems logical but a reference is always good)

**Comment by flogross89** [comment]
> Can this be merged before release?

We‘ll try, should be a comparably small change. Can you fix the merge conflict? Then we can start QA

**Comment by Fnni6** [comment]
It is blank indeed as seen in some online reference I found.

IMG_0018

**Comment by flogross89** [comment]
also needs lint fix

**Comment by Steveveepee** [comment]
I posted a short video of the functioning of that display ages ago. Apart from the off not being off, it's pretty spot on. 

Thanks all for pushing this through.

**Comment by flogross89** [comment]
image
seems the font doesn't load when switching from OFF to something else. I'll investigate

**Comment by flogross89** [comment]
I changed the colors, and also moved some of the code to the common BAT component, which removed the above bug
image
image

It's now completely off when on OFF

**Comment by Maximilian-Reuter** [comment]
awesome I was pretty sure that hadnt been done yet but wasnt sure

**Comment by Maximilian-Reuter** [comment]
> I changed the colors, and also moved some of the code to the common BAT component, which removed the above bug image image
> 
> It's now completely off when on OFF

wouldnt it make more sense to have the style or at least color be controlable by the instantiation ? cause pretty sure the old color was perfect for the 320 which you now changed as well right by doing it in common ?

**Comment by flogross89** [comment]
> > I changed the colors, and also moved some of the code to the common BAT component, which removed the above bug image image
> > It's now completely off when on OFF
> 
> wouldnt it make more sense to have the style or at least color be controlable by the instantiation ? cause pretty sure the old color was perfect for the 320 which you now changed as well right by doing it in common ?

The color was just changed for the a380x, they both have their own style.scss in the BAT folder

**Comment by Maximilian-Reuter** [comment]
> > > I changed the colors, and also moved some of the code to the common BAT component, which removed the above bug image image
> > > It's now completely off when on OFF
> > 
> > 
> > wouldnt it make more sense to have the style or at least color be controlable by the instantiation ? cause pretty sure the old color was perfect for the 320 which you now changed as well right by doing it in common ?
> 
> The color was just changed for the a380x, they both have their own style.scss in the BAT folder

ignore me i cant read

**Review by flogross89 — APPROVED** [review]

**Comment by tylerclt** [comment]
Quality Assurance Trainee Report

Discord Username: tylerb780 
Object of testing : #8833 
Aircraft: A32NX 
Tier of Testing: 1
Date: 23/10/2024 

Testing Process: 
1. Loaded into the A32NX cold and dark 
2. Verified no regressions (batt display still showed a value when the aircraft was not powered).
3. Made sure it shows values when switching to any non-OFF position and verified the BAT displays update one digit at a time

Testing Results: 
Passed  

Conclusions: 
Works as intended

Media:
The first image is with no aircraft power, the second image is with the aircraft powered.

**Pull request merged**